kitchen.yml

```diff
@@ -12,13 +12,16 @@ verifier:
 
 platforms:
   - name: amazonlinux-2
-  - name: centos-6
   - name: centos-7
+  - name: centos-8
   - name: debian-9
   - name: debian-10
   - name: freebsd-11
-  - name: fedora-29
+  - name: freebsd-12
+  - name: fedora-30
+  - name: fedora-31
   - name: opensuse-leap-42
```

Comment:
```suggestion
```

Reply:
I'm open to removing it if others agree, only had it in for support of latest OS plus one previous.

Reply:
Remove it :+1: